docs/_documentations/mdt-vsc-usingadifferenttemplate.md

```diff
@@ -22,3 +22,12 @@ When you create a new project, you can choose from the default set of templates
 ```
 
 After you have edited and saved this file, restart Codewind.
```

Comment:
The above info is outdated and should be removed, the new UI replaces editing that file.